docs/content/preview/yugabyte-platform/administer-yugabyte-platform/high-availability.md

```diff
@@ -112,61 +114,107 @@ After the active instance has been configured, you can configure one or more sta
 
 Your standby instance is now configured.
 
+### Verify HA
+
+To confirm communication between the active and standby, you can do the following:
+
+- Click **Make Active** on the standby (but don't promote it). You should see a list of available backups that you can restore from.
+- Verify that Prometheus on the standby is able to see similar metrics to the active. Navigate to `http://<STANDBY_IP>:9090/targets`; the federate target should have a status of UP, and the endpoint should match the active instance IP address.
+
+    ![Verify Prometheus](/images/yp/high-availability/ha-prometheus.png)
+
+- Navigate to **Profile > Releases** and verify that the standby has all the database releases that are in use by universes, and that the releases are listed as Active. To discover all the database releases that are in use by universes, you can view the **Dashboard** page.
+
+    If releases are missing, follow the instructions in [How to configure YugabyteDB Anywhere to provide Older, Hotfix, or Debug Builds](https://support.yugabyte.com/hc/en-us/articles/360054421952-How-to-configure-YugabyteDB-Anywhere-to-provide-Older-Hotfix-or-Debug-Builds).
+
+If the YBA instances are configured to use the HTTPS protocol and you are having problems, verify that certificates and ports are set up correctly. If the issue persists, consider relaxing the certificate validation requirements as a workaround, by enabling the [runtime configuration](../manage-runtime-config/) `yb.ha.ws.ssl.loose.acceptAnyCertificate` (set the flag to `true`).
+
```

Comment:
Can we add this section, which we added to the internal doc recently

Load balancer setup

To set up a single URL that points to the current primary even after a switchover/failover, it is recommended to set up an Application Load Balancer with the health check URL for each instance set to https://<instance IP/DNS>/ha_leader. Note that you may need to include the Load Balancer URL to the “Support Origin URL” as documented in this [support KB](https://support.yugabyte.com/hc/en-us/articles/360060669972-Error-Invalid-CORS-request-in-logs-when-logging-into-Yugabyte-Platform-Yugaware-UI).